### ISSUE TITLE
Robo Recall stays unresponsive

### ISSUE THREAD

**Issue by spin360** [LibreVR/RoboRevive#28]
So, I just wanted to demo the Vive after not using it for some time, but Robo Recall didn't want to work for me.

I tried everything from reinstalling SteamVR (I had the beta with home) to reinstalling Robo Recall completely, disabling direct mode and using admin mode on Steam, but to no avail.

When I launch it through the HMD, the window stays black. When I launch it from the desktop icon, SteamVR shows "(unresponsive) Robo Recall" (in red) as the running VR app.

I'm out of ideas and I'll demo it again in a few hours, so it kind of sucks. Not sure if the problem is with RoboRevive or Robo Recall.
Also tried the non beta version of steamVR and uninstalled the Revive dashboard.
![unresponsive(https://cloud.githubusercontent.com/assets/23230728/26454854/903dd5f6-4168-11e7-86d8-7f97cb6cf2a8.PNG)

**Comment by Armada651** [comment]
Have you tried starting Robo Recall through the desktop shortcut?

**Comment by spin360** [comment]
Yup (as I said)
Only then does it show unresponsive. When I launch it through SteamVR it doesn't say unresponsive but it still is. Waited for like half an hour too - window stays black.

**Comment by Armada651** [comment]
Ah now I see you did do that. Yeah, there seems to be a group of people who can't start RoboRevive. You could try starting it through the regular Revive Dashboard, but it will not use the RoboRevive mod in that case.

**Comment by spin360** [comment]
It's weird, I installed revive several times now but the tab stays empty. 

Also reinstalled oculus home and the game in default folder, nothing changed. Reinstalled all drivers and older Nvidia drivers, same problem.

**Comment by Armada651** [comment]
Sounds like a deeper problem with your Oculus Home installation then. Are you sure you installed the latest version? The tab should never be empty.

**Comment by spin360** [comment]
Yup I'm absolutely sure, since reinstalled everything from steam, to the game, to revive and oculus home. Before reinstalling I ran CCleaner (reg  and Cache cleaner)

**Comment by Armada651** [comment]
I think CCleaner does more harm than good, but okay. Can you post your logs from `%APPDATA%\Revive`? Just enter that into the address bar and drag and drop the log files in the comment box here to upload them.

**Comment by spin360** [comment]
[ReviveInjector.txt](https://github.com/LibreVR/RoboRevive/files/1034547/ReviveInjector.txt)
[ReviveOverlay.txt](https://github.com/LibreVR/RoboRevive/files/1034548/ReviveOverlay.txt)


gladly!

Thanks a huge lot for the help :)

**Comment by Armada651** [comment]
Interesting bug in that log, a trailing slash is missing. That could potentially break the dashboard, though I'm not sure.

**Comment by spin360** [comment]
Hmmm, I don't really understand. Can I fix it somehow?

**Comment by spin360** [comment]
Alright I think I figured it out!

I got a new graphics card for my main PC (was using my laptop prior). Game ran fine, so I copied the save games that were still on the laptop, which kinda surprised me since I uninstalled everything. Was happy though, because I played a lot already.

Well the moment I copied the saves it stopped launching!
I now deleted the OdinSettings.sav in AppData\Local\RoboRecall\Saved\SaveGames
Working again!
My saves are not lost either, because it's the OdinSave1.sav that contains my save :D

**Comment by mamefan** [comment]
I have the same problem but didn't switch PCs or move saves around.

**Comment by spin360** [comment]
Try deleting the file I said. It didn't work for me on a single PC either, I just figured out the problem because I switched pc. It's working on both PCs now.

**Comment by PoloB49** [comment]
Dude... Thank you so much, I can't believe the amount of time I spent trying to resolve this issue. Thank you a lot for this, it works perfectly fine now
